sources/packages/api/src-sql/Institution/Create-Institution.sql

```diff
@@ -0,0 +1,45 @@
+CREATE TABLE IF NOT EXISTS institutions(
+  id SERIAL PRIMARY KEY, 
+  legal_operating_name VARCHAR(64) NOT NULL, 
```

Comment:
Maybe it would be better to check the length that these fields should have. Is 64 something present in the requirement?
If not, maybe @abschwenker could have a better idea.

Reply:
I'm not sure the max length BCeID could provide. I can reach out

Reply:
@abschwenker, it is not about BCeID itself, it is about all other text fields here.

Reply:
64 for field lengths is good